src/webapp/src/utilities/reporting.ts

```diff
@@ -0,0 +1,24 @@
+import * as uuid from "uuid"
+
+import Reporter, { HumbugConsent } from "@bugout/humbug"
+
+const HUMBUG_TOKEN = "3bafba59-f078-4004-ab12-fa27cf4885f1"
+
+const packagePath = process.cwd()
```

Comment:
I think it's better to get this from from [`__dirname`](https://nodejs.org/api/modules.html#modules_dirname) instead of `process.cwd()`.

`process.cwd()` is I think the directory from which the script was executed. It may be different than the project root directory.